README.md

```diff
@@ -24,3 +24,14 @@ HTCondor-CE RPMs are available from the following locations:
 
 - HTCondor Yum repositories: https://research.cs.wisc.edu/htcondor/yum/
 - OSG Yum repositories: https://opensciencegrid.org/docs/common/yum/
+
+Versioning
+----------
+
+At any given time, two versions of HTCondor-CE are maintained, a stable and a development version.
+In this repository, the `master` branch contains the latest version of HTCondor-CE (i.e. development) while the `stable`
+branch contains the previous version.
+As of September 2019, we maintain two different versions of HTCondor-CE:
```

Comment:
```suggestion
```
With your new sentence, this is a dupe.

Reply:
Alternatively, phrase it as
"As of September 2019, the two different versions of HTCondor-CE we maintain are:"